src/components/IPScan/Summary/IPScanTitle/index.tsx

```diff
@@ -35,62 +35,63 @@ const IPScanTitle = ({
   editable = true,
 }: Props) => {
   const [title, setTitle] = useState(localTitle || '');
-  const [readable, setReadable] = useState(true);
+  const [readOnly, setReadOnly] = useState(true);
```

Comment:
Otherwise the `input` is created read-only and we still have to click the pencil icon to edit the title:

```suggestion
  const [readOnly, setReadOnly] = useState(!editable);
```